index.html

```diff
@@ -0,0 +1,122 @@
+                        <li><a href="about.html/#past-events">Past Events</a></li>
+                    </ul>
+                </nav>
+            </div>
+        </div>
+    </section>
+    <div class="mobile-menu-pane">
+        <i class="fa-solid fa-xmark" onclick="removeStuffs()"></i>
+        <nav>
+            <ul class="mobile-nav-items">
+                <li><a href="index.html" onclick="removeStuffs()" class="nav-item active">Home</a></li>
+                <li><a href="about.html" onclick="removeStuffs()" class="nav-item">About Conference</a></li>
+                <li><a href="about.html/#past-events" onclick="removeStuffs()" class="nav-item">Past Events</a></li>
+            </ul>
+        </nav>
+    </div>
+    <section class="intro container">
+        <i class="fa-solid fa-bars hamburger"></i>
+        <p class="intro-Text">"GDIW 2022"</p>
+        <h1>Glocalizing Ghana's Digital <br> & Innovation Ecosystem: Opportunities <br> for Economic Development</h1>
+        <div class="border-white intro-desc">
+            <p>A trilateral program to celebrate the accomplishments of Ghana's digital innovation ecosystem and build <br> momentum towards a common vision for its future development.</p>
+        </div>
+        <p class="venue"><span class="dates">2022.10.26(WED) ~ 28(FRI)</span> <br> @Accra International Conference Centre</p>
+    </section>
+    <section id="highlights" class="container">
+        <h3 class="highlights-title">Main Program</h3>
+        <hr class="divider">
+        <div class="activity-items">
+            <div class="box-h-item flex-activities">
+                <img src="img/conference.png" alt="Panel Discussion Icon">
+                <h3>Panel Discustions</h3>
+                <div>
+                    <p>Responsible AI and ethics in response to Fourth Industrial Revolution</p>
+                </div>
+            </div>
+            <div class="box-h-item flex-activities">
+                <img src="img/workshop.png" alt="Workshop Icon">
+                <h3>Workshops</h3>
+                <div>
+                    <p>Extrapreneuring Our Way into Digital Transformation</p>
+                </div>
+            </div>
+            <div class="box-h-item flex-activities">
+                <img src="img/speech-bubble.png" alt="Forum Icon">
+                <h3>Panel Discustions</h3>
+                <div>
+                    <p>Responsible AI and ethics in response to Fourth Industrial Revolution</p>
+                </div>
+            </div>
+            <div class="box-h-item flex-activities">
+                <img src="img/exhibition.png" alt="Exhibition Icon">
+                <h3>Training Session</h3>
+                <div>
+                    <p>Responsible AI and ethics in response to Fourth Industrial Revolution</p>
+                </div>
+            </div>
+            <div class="box-h-item flex-activities">
+                <img src="img/fireball.png" alt="Fireball Icon">
```

Comment:
[OPTIONAL] The `main program` section can also be displayed dynamically.
Kindly check and make sure that this section follows the design dimensions ( like margins and paddings) as there are some discrepancies at the moment as indicated in the images below


Your mobile design             |  The actual design
:-------------------------:|:-------------------------:
![Screenshot from 2022-10-19 08-43-06](https://user-images.githubusercontent.com/65233860/196659906-058783d1-2dc3-453b-a2ef-a5159fc35dc4.png)  |  ![Screenshot from 2022-10-19 09-52-13](https://user-images.githubusercontent.com/65233860/196659957-7fe3c2f7-3447-4171-94e4-ae903bda7245.png)


Your desktop design             |  The actual design
:-------------------------:|:-------------------------:
![Screenshot from 2022-10-19 08-41-09](https://user-images.githubusercontent.com/65233860/196660376-09a4bae1-04a0-462e-a89a-58fab6d8b4c8.png)  |  ![Screenshot from 2022-10-19 09-53-50](https://user-images.githubusercontent.com/65233860/196660435-dc339f9f-0436-4415-9abd-00e028a8ecf5.png)